src/components/LoansByCategory/FeaturedHeroLoanWrapper.vue

```diff
@@ -94,24 +95,26 @@ export default {
 	},
 	apollo: {
 		preFetch(config, client) {
-			// check if hero channel experiment is enabled
-			return getExperimentSettingAsync(client, heroChannelExpKey)
-				.then(({ enabled }) => {
-					// use ml query to get loan channel to use if experiment is enabled
-					if (enabled) {
-						return getHeroChannelAsync(client);
-					}
-					// otherwise use default query
-					return getFeaturedLoanChannelAsync(client);
```

Comment:
Now that the ML served hero channel is the default, we no longer fallback to the Kiva settings options. Rather we will fallback to the hard coded channel if `ml` doesn't resolve one.